### ISSUE TITLE
flag active/inactive (or delete?)

### ISSUE THREAD

**Issue by Trevoke** [Trevoke/dfworldgen#11]
Since we can't edit a parameter set, there'll be some that will get deleted and reposted. Do we want to flag them active/inactive or just up and delete?

**Comment by yamamushi** [comment]
Maybe flag them with updates, like :

"This parameter set has been updated by the uploader - Click Here"

Something along those lines?

**Comment by Trevoke** [comment]
Ah.. Interesting. So maybe what we want is really a clone feature that will copy everything to a new one and then disable voting on the old one, providing a link to the new one?

**Comment by yamamushi** [comment]
I don't think we need to copy it over, rather the uploader should have the option of checking a box or something that says "is this an update to a previous parameter set?" and then it will let them select one that is a branch of.

**Comment by Trevoke** [comment]
Why give the user more work?
This makes me think we probably need a list of "my submissions" or
something.

On Sun, Feb 26, 2012 at 10:12, Jonathan Rumion <
reply@reply.github.com

**Comment by yamamushi** [comment]
Yeah, definitely something along those lines. Or even a link on a page where you're viewing a parameter that says:

"Submit alternate version of this worldgen"

And if you're not the original submitter it would be submitted as a "branch" or "modified" version of that one, or if you are the original one the application would detect that you are the logged in user and submit it as an update or modification or branch depending on how the user wanted it to be submitted as.

**Comment by Trevoke** [comment]
Wow, you want to reimplement github :-)
At this point, it might actually be easier to implement that with git in
the backend :p
I jest, but yeah, that's a whole different feature. Feel free to open an
issue with your vision for that, but I'm thinking of a level one thing -
that we can do and release and then improve on.

On Sun, Feb 26, 2012 at 10:50, Jonathan Rumion <
reply@reply.github.com

**Comment by yamamushi** [comment]
Haha, it's a long term goal of course. But as for an initial release, we really shouldn't have to worry about these types of features.
